Makefile

```diff
@@ -1,48 +1,41 @@
 all: install
 
 ifeq ($(OS), Windows_NT)
-	Platform := "Windows"
+	Platform=Msys
+install:
+	mkdir $(USERPROFILE)\.cache 2> /dev/null
+	cp ani-cli $(WINDIR)/system32/ani-cli
+uninstall:
+	rm -rf $WINDIR/system32/ani-cli
```

Comment:
It seems better practice to seperate platform detection from platform specific `install` and `uninstall`

Reply:
Yea thats the thing,it doesnt work if i seperate install uninstall from the platform detection condition,this happens only to windows, it will only output "Platform=Msys"

Reply:
nvm solved